Update local-emulator.md

AndyCross opened the pull request:
The reference in the proposed change should either be "Cosmos DB in the cloud" or "Document DB in the cloud"; presumably the former for consistency with the rest of the document.

azurecla commented:
Hi __@abhiver__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
Update local-emulator.md

The reference in the proposed change should either be "Cosmos DB in the cloud" or "Document DB in the cloud"; presumably the former for consistency with the rest of the document.
